app/experimenter/docs/adrs/0006-doc-hub-repo.md

```diff
@@ -0,0 +1,98 @@
+# Location for the new Experiment "Docs Hub" codebase + docs
+
+- Status: **draft**
+- Deciders: Kate Hudson, Jared Lockhart, Lauren Zugai, Tim Smith, Les Orchard, Jody Heavener
+- Date: 2021-03-12
+
+## Context and Problem Statement
+
+ADR #5, ["Use Docusaurus + GH Pages for the Nimbus User Doc Hub"](https://github.com/mozilla/experimenter/blob/main/app/experimenter/docs/adrs/0005-doc-hub.md), determined that our approach to setting up a new central location for Experimenter/Nimbus and related project documentation would use [Docusaurus](https://v2.docusaurus.io/), a static site generator, with the static files being served from [GitHub Pages](https://pages.github.com/). While we knew we would be introducing a new set of code we didn't explicitly decide where it would live and what the process for deploying would look like, beyond suggesting it could live in the existing Experimenter repository.
+
+In investigating how to set all of this up we realized there would be certain caveats to hosting and deploying this code from the Experimenter repository, and this lead to the consideration of using a dedicated repository. This ADR outlines our options and end decision.
+
+## Decision Drivers
+
+- How much setup work is involved, both for the code _and_ for integrations or other non-code processes?
+- How maintainable is this setup, both now and in years to come?
```

Comment:
- How much overhead and complexity is involved in maintaining this setup / keeping our processes and tools in sync with experimenter